src/languages/php.js

```diff
@@ -394,6 +393,43 @@ export default function(hljs) {
     ]
   };
 
+  const PARAMS_MODE = {
+    relevance: 0,
+    begin: /\(/,
+    end: /\)/,
+    keywords: KEYWORDS,
+    contains: [
+      {
+        scope: 'attr',
+        match: regex.concat(IDENT_RE, regex.lookahead(':'), regex.lookahead(/(?!::)/)),
+      },
```

Comment:
```suggestion
      NAMED_ARGUMENT,
```

abstract this, like the others please.

Reply:
Done